base/process.jl

```diff
@@ -311,7 +311,9 @@ macro setup_stdio()
         in,out,err = stdios
         if isa(stdios[1], Pipe)
             if stdios[1].handle == C_NULL
```

Comment:
As per our discussion, the C_NULL here should be fine as this way you get an error in the use-after-close case whereas otherwise it would silently pass `/dev/null`.